bundles/org.openhab.core.io.bin2json/src/main/java/org/openhab/core/io/bin2json/ConversionException.java

```diff
@@ -0,0 +1,42 @@
+/**
+ * Copyright (c) 2010-2019 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.core.io.bin2json;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+
+/**
+ * {@link ConversionException} generic exception for errors which occurs during conversion.
+ *
+ * @author Pauli Anttila - Initial contribution
+ */
+@NonNullByDefault
+public class ConversionException extends Exception {
+
+    private static final long serialVersionUID = 1L;
```

Comment:
Just a curiosity, why do you think that generated is better than just start from 0 or 1? I mean if someone will make changes to class which are not backward compatible from serialisation point of view, serialVersionUID need to be manually changed by the developer as serialVersionUID is anyhow explicitly defined.

Reply:
I never ever gave much thoughts on this myself, I must admit. I always used the generation feature of Eclipse for it. But I cannot give an answer, why this would be better than just 0...
I have also never ever experienced that anyone changed the serialVersionUID when doing changes to the class - in theory, we should enforce this, I guess. But as we don't use Java serialisation anywhere in openHAB, I think that it really hardly matters at all...

Reply:
Alright. It is clear to me now. We do not care very much - which is okay - because we do not need it.

@paulianttila Up to you if you like to change it or not.